lib/diaspora_federation/entities/relayable_retraction.rb

```diff
@@ -46,7 +46,45 @@ class RelayableRetraction < Entity
       #   This signature is mandatory only when federation from the subscriber to an upstream
       #   author is done.
       #   @return [String] target author signature
-      property :target_author_signature
+      property :target_author_signature, default: nil
+
+      # Generates XML and updates signatures
+      # @see Entity#to_xml
+      # @return [Nokogiri::XML::Element] root element containing properties as child elements
+      def to_xml
+        entity_xml.tap do |xml|
+          hash = to_h
+          self.class.update_signatures!(hash)
```

Comment:
in `Relayable` it is also `Relayable.update_signatures!(hash)`. I think if `self.class` always is the same, we should use the class name directly.